apis/r/R/utils.R

```diff
@@ -46,6 +46,12 @@ null <- function(...) {
   return(NULL)
 }
 
+random_name <- function(length = 5L, chars = letters, ...) {
+  stopifnot(is_integerish(length, n = 1L), is.character(chars))
```

Comment:
It's nicer with error messages ie

```r
stopifnot("'length' must be integer" = is_integerish(length, n = 1L), 
                  '"chars" must be character" = is.character(chars))
```

or alike but the current form is of course fully functional.

Reply:
I've left it without messages because this is an internal, non-exported function

Reply:
I disagree.  The code I suggested is also a comment and an explanation.  Explicit beats implicit.

But you and I will never agree on R stylistic matters but as long as the right result comes out we don't need to split hairs.

Reply:
Error messages added in 12dda97